### DISCUSSION TITLE
New set top box: changing remote ?

### DISCUSSION THREAD

**Discussion by waldorf-statler** [GerryDazoo/Slinger#652] [General]
I wonder if anyone has been able to introduce remote support for a new box (that was not previously loaded into the slingbox configuration), and if so, any pointers on how to do this? I was looking into the various similar discussions so far, but could not see a sufficiently clear answer. I realize there are alternative ways to do this, but it would be nice not to buy yet another box just for the IR control.
I also have the additional problem that I have not been able to locate a bin file either (or have the data to make one), but before going too far down that rathole, just wanted to check that it is possible to add something new. Any help, comments appreciated!

**Comment by garabik** [comment]
IMHO depends on the Slingbox model. Definitely possible with a factory-reset Solo and Pro HD (the latter I unintentionally tested several times) and the original Slingplayer 1.5 (_not_ 2.x) for Windows. Make sure you are **not** connected to the internet during the setup, and it sometimes takes several tries (timeout at the end of the process). Even using a custom .bin file for a custom (i.e. non supported) remote works (but you must already have the .bin file, e.g. from the JP1 forum).

Useful links:

- how to set up a custom remote, the exacts steps and menus are different for Pro HD/Solo and Slingplayer 1.5, but close enough to be able to guide you: http://www.hifi-remote.com/wiki/index.php/How_to_Install_a_Custom_Remote_onto_the_Slingbox
- Many ready-to-use custom remote .bin files: http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=87 (even if your box is not listed, another one might be compatible enough with only some keys missing or rearranged - the real problem is finding which one)
- https://slingbox.fandom.com/wiki/How_to_create_custom_remote_upgrade

**Reply by waldorf-statler** [reply]
Thanks a lot, really loads of good information there, that is really helpful. 

My model is an M1. There are definitely .bin files I could try, even if nothing is guaranteed as no exact match. 

So my main concern now is whether the machine could be recovered after a factory reset. I wonder if anyone has been able to do that (_set up an M1 for use with slinger from scratch after a factory reset_). 

Or if not, perhaps I could try eBay :)

**Reply by BroozerL** [reply]
My Slingbox is a ProHD.  My interest is OTA streaming.  Of course, the Slingbox apps are not (currently) set up to scan for ATSC; hence, I purchased an IVIEW 3300STB via Amazon.  JP1 has a .bin for the IVIEW 3500.  I followed the above instructions and successfully got the .bin added via SlingPlayer 1.5.1.342.  And the remote works good enough.  🙂

How do I install/convert the C2010_PL.bin to function on the Slinger server & player?

**Reply by garabik** [reply]
> How do I install/convert the C2010_PL.bin to function on the Slinger server & player?

That should work out of the box, the .bin file is stored in the slingbox itself. You might only want to rearrange the buttons / create nicer remote, by editing the html file referenced in the `include` directive in the `config.ini` file.

**Reply by Cibinium** [reply]
Hi All, 

I have Slingbox 500 and I can't add any remote (any bin file), the SlingPlayer Desktop can't do to finish the setup because for 500 it's need the original Sling servers, I think, for Slingbox M1 I was able to do with SlingPlayer Desktop.

Any tips or tricks ?

Thank you.

**Reply by RayMan412** [reply]
Didn't the MAC version of SlingPlayer allow the Digital OTA Scan?
